package.json

```diff
@@ -21,6 +21,8 @@
     "server:ctr": "node src/server/generateServerKey.js",
     "start": "NODE_ENV=development webpack serve --config config/dev.webpack.config.js",
     "start:beta": "NODE_ENV=development BETA=true webpack serve --config config/dev.webpack.config.js",
+    "start:proxy": "NODE_ENV=development API_ENDOINT=https://cloud.redhat.com webpack serve --config config/dev.webpack.config.js",
+    "start:stage": "NODE_ENV=development API_ENDOINT=https://console.stage.redhat.com webpack serve --config config/dev.webpack.config.js",
```

Comment:
No need to add these, if you want to run it in prod env you can do that by calling `ENVIRONMENT=prod npm run start`
```suggestion
```